src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java

```diff
@@ -176,6 +182,37 @@ public AboutToStartOrSubmitCallbackResponse prepopulateFL401CaseDetails(
         @RequestBody CallbackRequest callbackRequest
     ) {
         if (authorisationService.isAuthorized(authorisation,s2sToken)) {
+            CaseData caseData = objectMapper.convertValue(
+                callbackRequest.getCaseDetails().getData(),
+                CaseData.class
+            );
+
+            String caseType = CaseUtils.getCaseTypeOfApplication(caseData);
+            String selectedOrder = caseData.getCreateSelectOrderOptions().getDisplayedValue();
+            List<String> errorList = new ArrayList<>();
+
+            if (Objects.equals(caseType, "C100") && (!CreateSelectOrderOptionsEnum.blankOrderOrDirections.getDisplayedValue().equals(selectedOrder)
```

Comment:
Can you please refer how we are checking for caseType of application? and do the same as discussed